content/swiftui/concepts/views/terms/rectangle/rectangle.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Rectangle'
+Description: 'A rectangular shape aligned inside the frame of the view containing it.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Rectangle'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`rectangle`** A rectangular shape aligned inside the frame of the view containing it.
+
+## Syntax
+
+```pseudo
+var body: some View {
+  Rectangle()
+}
+```
+
+SwiftUI has several built-in shapes, such as `Rectangle`, circles, and capsules, and you can create, color, and place each shape as needed.
```

Comment:
Keep consistent here, either name all of them uppercased and in backticks (referencing the view in SwiftUI syntax), or all without backticks and lowercased (plain English).

Reply:
```suggestion
SwiftUI has several built-in shapes, such as a rectangle, circle, and capsule. The color and placement of each shape can be modified as needed.
```

Reply:
I changed it !